docs/sources/whatsnew/whats-new-next/index.md

```diff
@@ -128,3 +128,12 @@ _Available in private preview in Grafana Cloud_
 The Grafana Assume Role authentication provider lets Grafana Cloud users of the CloudWatch data source authenticate with AWS without having to create and maintain long term AWS Users. Using the new assume role authentication method, you no longer have to rotate access and secret keys in your CloudWatch data source. Instead, Grafana Cloud users can create an identity access and management (IAM) role that has a trust relationship with Grafana's AWS account; Grafana's AWS account will then use AWS Secure Token Service (STS) to create temporary credentials to access the user's AWS data.
 
 To learn more, refer to the [CloudWatch authentication documentation](/docs/grafana/next/datasources/aws-cloudwatch/aws-authentication).
+
+## Permission validation on custom role creation and update
+
+<!-- Mihaly Gyongyosi -->
+<!-- Enterprise -->
+
+_Generally available in Grafana Cloud_
```

Comment:
I might have misunderstood your message in Slack, but we do need this is with an Enterprise license? Or did you mean that's who's already got it but it will be generally available on Cloud?

Reply:
Apologies, there was a change few months ago that made RBAC available for all of the Cloud licenses, so it's available for all Cloud customers.